starknet-providers/src/jsonrpc/models/mod.rs

```diff
@@ -18,21 +18,15 @@ pub enum MaybePendingBlockWithTxHashes {
     PendingBlock(PendingBlockWithTxHashes),
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Deserialize)]
```

Comment:
There are many occurrences where the `Serialize` derive gets removed by this PR. Please don't do this. Technically we don't need serialization for the JSON-RPC client to work. However, as an SDK we need to consider use cases of dependent libs/apps, who are very likely to need serialization to function properly.

Reply:
Serialize trait derives added back.